lib/Hero/Content.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Button as ButtonComponent } from '../Button';
+
+const Content = ({
+  classNames = {},
+  title,
+  subTitle,
+  imageUrl,
+  hasOverlay,
+  NextImage,
+  Button = () => (
```

Comment:
Let's not set a default button component in here. Better to do that in the `stories/Hero.stories.jsx` file. Let's simply accept the `Button` prop and render it. If no `Button` prop is passed, then don't render the `Button` component at all.